packages/swapper/src/constants.ts

```diff
@@ -39,7 +39,7 @@ import { makeSwapErrorRight } from './utils'
 
 export const DEFAULT_GET_TRADE_QUOTE_POLLING_INTERVAL = 20_000
 export const QUOTE_TIMEOUT_MS = 60_000
-export const TRADE_POLL_INTERVAL_MILLISECONDS = 5_000
+export const TRADE_POLL_INTERVAL_MILLISECONDS = 1_000
```

Comment:
q: Isn't this too agressive? This will also apply to good ol' swapper and could easily produce XHR spew and perf downsides at best, rate-limits at worst

Reply:
Considering it has gcTime and staleTime, it's not going to fetch anymore than something every 10sec, but it if the action center catch a different status, it will be invalidated faster, which will kinda sync everything

It won't fetch more than something every 10sec, which is wrong (changing cache time to the actual poll interval to avoid rate limites/perf issues)

Reply:
I am confusion here, isn't that a 1 sec polling time from 5 before?

Also neither `staleTime` nor `gcTime` should affect fetching times since they're both set to the same `TRADE_POLL_INTERVAL_MILLISECONDS` i.e goes stale and garbage collects every sec

Reply:
Also can we make this `TRADE_STATUS_POLL_INTERVAL_MILLISECONDS` pls, as terminology here seems to imply this is the same as `DEFAULT_GET_TRADE_QUOTE_POLLING_INTERVAL` (a trade quote polling interval)